### PR TITLE
Allow spatie/invade 2.x

### PR DESCRIPTION
**Pull request by hamrak**
- [X] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

File: packages/support/src/Commands/Concerns/CanReadModelSchemas.php [hunk 1]
```diff
@@ -43,21 +44,21 @@ protected function guessBelongsToRelationshipName(AbstractAsset $column, string
     {
         /** @var Model $modelInstance */
         $modelInstance = app($model);
-        $modelInstanceReflection = invade($modelInstance);
+        $modelInstanceReflection = new ReflectionClass(invade($modelInstance));
```
Comment (danharrin) [inline packages/support/src/Commands/Concerns/CanReadModelSchemas.php]: Do we really need invade here?

```suggestion
        $modelInstanceReflection = new ReflectionClass($modelInstance);
```

Comment (hamrak) [reply packages/support/src/Commands/Concerns/CanReadModelSchemas.php]: yes, you right.

Comment (danharrin) [reply packages/support/src/Commands/Concerns/CanReadModelSchemas.php]: Did you test your previous code?

Comment (hamrak) [reply packages/support/src/Commands/Concerns/CanReadModelSchemas.php]:
```php
class Invader
{
    /** @var T */
    public object $obj;
    public ReflectionClass $reflected;

    /**
     * @param T $obj
     */
    public function __construct(object $obj)
    {
        $this->obj = $obj;
        $this->reflected = new ReflectionClass($obj);
    }
...
```

This was in version 1.1.1.

In 2.0 is this
```php
class Invader
{
    /**
     * @param T $obj
     */
    public function __construct(
        public object $obj
    ) {
    }
...
```

**Pull request merged**